pom.xml

```diff
@@ -0,0 +1,135 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>gov.usgs.ngwmn</groupId>
+	<artifactId>statistics</artifactId>
+	<name>statistics</name>
+	<description>Proof of concept to pull statistics out of monolithic NGWMN and use it for scientific review.</description>
+	<version>0.2.0</version>
```

Comment:
This should probably be 0.1.0-SNAPSHOT. We should follow are standard versioning process.